RefactorTest/Require.h

```diff
@@ -20,7 +20,8 @@ void require_equal(char const *file, unsigned line, T expected, U actual)
     if (expected != actual)
     {
         std::ostringstream message;
-        message << file << '(' << line << "): error: expected " << expected << ", got " << actual;
+        //message << file << '(' << line << "): error: expected " << expected << ", got " << actual;
```

Comment:
The only place where `enum class` is used is in `RenameCpp11.cpp` and in all cases the enum is cast to an int before calling `require_equal`

Reply:
For your new tests cast to `int` or move the enum to a namespace scope where a stream insertion operator can be defined would be the right fix instead of changing the entire `require_equal` mechanism.

Reply:
I confused about this comment, on `master` branch, when I build `refactor-test-suite`, I get the errors below which states the lines that I've never changed:

.../RefactorTest/Require.h:23:65: error: call to function 'operator<<' that is neither visible in the template definition nor found by argument-dependent lookup
        message << file << '(' << line << "): error: expected " << expected << ", got " << actual;
                                                                ^
.../RefactorTest/RenameCpp20.cpp:73:9: note: in instantiation of function template specialization 'require_equal<std::strong_ordering, std::strong_ordering>' requested here
        REQUIRE_EQUAL(std::strong_ordering::equal, x <=> x);

Reply:
Ah, OK... I thought I had added stream insertion operators for those... linux CI build is compiling without errors.

Reply:
What version compiler are you using?

Reply:
google says ubuntu 22.04.3 LTS is using gcc-11, but I can't reproduce your error on compiler explorer:
https://godbolt.org/z/36Y6vxeTn

Reply:
Please revert this change.